dds2val/src/README.md

```diff
@@ -0,0 +1,24 @@
+# DDS Feeder
+
+## How to build
+### local build
+
+1. ```python3 -m venv env && source env/bin/activate```
+2. ```pip install -r requirements.txt```
+3. ```./idls/generate_py_dataclass.sh```
+
+### build image (suggested)
+
+1. ```docker build -f Dockerfile --progress=plain --build-arg TARGETPLATFORM=linux/amd64 -t ddsfeeder:latest .```
+
+### KML replay
+
+1. ```python3 -m venv env && source env/bin/activate```
+2. ```pip install -r requirements-kml.txt```
+
+## How to run
+Choose from local build or contanerization via docker.
+These steps are necessary:
+1. Run an instance of databroker aka: ```docker run -it --rm --net=host ghcr.io/eclipse/kuksa.val/databroker:master```
+2. Start the KML replay with an active local python virtual environment: ```python3 dds_kmlreplay.py directions.kml```
+3. Start the DDS feeder with either: ```docker run --rm -it --net=host ddsfeeder:latest``` or with an active local python virtual environment: ```python3 ddsfeeder.py```
```

Comment:
I tried to run both variants (built docker container and directly with python), got similar errors:
```

erik@debian3:~/kuksa.val.feeders/dds2val/src$ !4778
docker run --rm -it --net=host ddsfeeder:latest
Traceback (most recent call last):
  File "ddsfeeder.py", line 41, in <module>
ImportError: attempted relative import with no known parent package
[8] Failed to execute script 'ddsfeeder' due to unhandled exception!

```

Reply:
I am no python import expert maybe some help from @romainletendart ;) ? If you have time :D Else maybe some thoughts from @AkhilTThomas ?

Reply:
Did you find a solution that satisfies both tests and docker? I do not know if restructuring like in dbcfeeder so that only the "main file" is in the top repo would help.

Reply:
Now it is working for me if I am in the dds2val directory and execute tests and ddsprovider.py or docker build from there